### PR TITLE
python : getting started use raw paho

### PR DESCRIPTION
**Pull request by olivakar**

### CODE REVIEW

File: scenarios/getting_started/python/getting_started.py [hunk 1]
```diff
@@ -9,56 +16,89 @@ def on_connect(client, _userdata, _flags, rc):
...
+    # SUBSCRIBE
     (_subscribe_result, subscribe_mid) = client.subscribe("sample/+")
     print(f"Sending subscribe requestor topic \"{topic}\" with message id {subscribe_mid}")
 
 def on_subscribe(client, _userdata, mid, _granted_qos):    
     print(f"Subscribe for message id {mid} acknowledged by MQTT broker")
+    # PUBLISH
     topic = "sample/topic1"
     payload = "hello world!"
     publish_result = client.publish(topic, payload)
```
Comment (cartertinney) [inline scenarios/getting_started/python/getting_started.py:28]: This is likely inadvisable due to the thread model of Paho. I recommend instead triggering a notification of some kind (Condition, Event, etc.) and then responding to that notification elsewhere.

The contents of a callback are always invoked on the Paho thread. Invoking Paho on its own background thread likely could lead to some issues, and additionally, it just muddies the readability of the sample.

File: scenarios/getting_started/python/getting_started.py [hunk 2]
```diff
@@ -9,56 +16,89 @@ def on_connect(client, _userdata, _flags, rc):
...
-client.on_publish = on_publish
-client.on_subscribe = on_subscribe
+# CONNECT
+print("{}: Starting connection".format(client_id))
+hostname = connection_settings['MQTT_HOST_NAME']
+port = connection_settings['MQTT_TCP_PORT']
+keepalive = connection_settings["MQTT_KEEP_ALIVE_IN_SECONDS"]
+mqtt_client.connect(hostname, port,keepalive)
+print("Starting network loop")
+mqtt_client.loop_forever()
```
Comment (cartertinney) [inline scenarios/getting_started/python/getting_started.py:103]: subscribes and publishes should happen here

**Review by cartertinney — CHANGES_REQUESTED** [review]

File: mqttclients/python/connectionsettings/connection_settings.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    'MQTT_HOST_NAME',
+    'MQTT_TCP_PORT',
+    'MQTT_USE_TLS',
+    'MQTT_CLEAN_SESSION',
+    'MQTT_KEEP_ALIVE_IN_SECONDS',
+    'MQTT_CLIENT_ID',
+    'MQTT_USERNAME',
+    'MQTT_PASSWORD',
+    'MQTT_CA_FILE',
+    'MQTT_CA_PATH',
```
Comment (vaavva) [inline mqttclients/python/connectionsettings/connection_settings.py:37]: This can be removed after my PR removed it!

**Review by cartertinney — APPROVED** [review]

**Review by cartertinney — APPROVED** [review]

**Review by cartertinney — APPROVED** [review]

**Review by vaavva — APPROVED** [review]

File: scenarios/getting_started/python/getting_started.py (hunk 2 above)
Comment (olivakar) [reply scenarios/getting_started/python/getting_started.py:187]: all callback set conditions now

**Pull request merged**